.gitignore

```diff
@@ -5,6 +5,9 @@ dist/
 ./movie-app/dist
```

Comment:
Unit tests coverage is insufficient. I've run coverage report for all project files and see the following picture:
```
29.87% Statements 23/77
50% Branches 12/24
42.86% Functions 12/28
30.67% Lines 23/75
```